CMakeLists.txt

```diff
@@ -9,12 +9,13 @@ include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 
 set(UTILS_SCRIPTS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/scripts)
 if (DTC)
-  add_custom_target(
-    fix_dtb
-    DEPENDS ${ZEPHYR_FINAL_EXECUTABLE}
-    COMMAND ${UTILS_SCRIPTS_DIR}/fix_dtb.sh ${UTILS_SCRIPTS_DIR}/gicv2.tmpl ${ZEPHYR_BINARY_DIR}/virt_gicv2.dtb ${ZEPHYR_BINARY_DIR}/zephyr.bin)
+    add_custom_target(
+        fix_dtb
+        DEPENDS ${ZEPHYR_FINAL_EXECUTABLE}
+        COMMAND ${UTILS_SCRIPTS_DIR}/fix_dtb.sh ${UTILS_SCRIPTS_DIR}/gicv${QEMU_GIC_VERSION}.tmpl ${ZEPHYR_BINARY_DIR}/virt_gicv${QEMU_GIC_VERSION}.dtb ${ZEPHYR_BINARY_DIR}/zephyr.bin
+    )
 else()
-  message(WARNING "No dtc program")
+    message(WARNING "No dtc progren")
```

Comment:
progren -> program. 
Also - do we really need this intendation changes?

Reply:
This is misprint

Reply:
please remove it from commit